src/components/layout/DashboardLayout.tsx

```diff
@@ -1,57 +1,53 @@
-import { Cog6ToothIcon, HomeIcon, SquaresPlusIcon, UsersIcon } from '@heroicons/react/24/outline';
-import { Outlet, useMatches, type UIMatch } from 'react-router-dom';
+import { RiMenuLine, RiMoonLine, RiSunLine } from '@remixicon/react';
+import { Link, Outlet, useMatches, type UIMatch } from 'react-router-dom';
+
+import { useTheme } from '@/providers';
+import { useMediaQuery } from '@/hooks';
+import { cn } from '@/lib';
 
 import {
+    Avatar,
+    AvatarFallback,
+    BreadcrumbElipssis,
     BreadcrumbItem,
+    BreadcrumbLink,
+    BreadcrumbList,
+    BreadcrumbPage,
+    BreadcrumbSeparator,
     Breadcrumbs,
-    Sidebar,
-    SidebarContent,
-    SidebarLink,
-    SidebarSection,
+    Button,
+    Dropdown,
+    DropdownContent,
+    DropdownGroup,
+    DropdownItem,
+    DropdownTrigger,
 } from '../ui';
 
-const navigation = [
+const NAVIGATION = [
+    {
+        name: 'Home',
+        to: '/dashboard/home',
+    },
     {
-        name: 'DASHBOARD',
-        links: [
-            {
-                name: 'Home',
-                href: '/dashboard/home',
-                icon: <HomeIcon />,
-            },
-            {
-                name: 'Model Registry',
-                href: '/dashboard/models',
-                icon: <SquaresPlusIcon />,
-            },
-        ],
+        name: 'Model Registry',
+        to: '/dashboard/models',
     },
     {
-        name: 'COLLABORATION',
-        links: [
-            {
-                name: 'Teams',
-                href: '/teams',
-                icon: <UsersIcon />,
-            },
-        ],
+        name: 'Teams',
+        to: '/dashboard/teams',
     },
     {
-        name: 'CUSTOMIZATION',
-        links: [
-            {
-                name: 'Settings',
-                href: '/settings',
-                icon: <Cog6ToothIcon />,
-            },
-        ],
+        name: 'Projects',
+        to: '/dashboard/projects',
     },
 ];
 
 export const DashboardLayout = () => {
-    // Am I a frontend engineer yet?
```

Comment:
Was this question ever resolved? 👀

Reply:
Yes it has! Turns out I am a professional front end engineer specializing in Streamlit and Azure 😂

Reply:
💀